automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainMultiGroupsSpec.scala

```diff
@@ -167,12 +167,14 @@ class AuthDomainMultiGroupsSpec extends FreeSpec with ParallelTestExecution with
         implicit val authToken: AuthToken = authTokenDefault
         withGroup("AuthDomain", List(user.email)) { groupOneName =>
           withGroup("AuthDomain", List(user.email)) { groupTwoName =>
-            withCleanUp {
-              withCleanBillingProject(user) { projectName =>
+            withCleanBillingProject(user) { projectName =>
+              withCleanUp {
```

Comment:
cleanup sequence is out of order. workspace delete should be done before billing project delete.